package.json

```diff
@@ -354,6 +354,7 @@
     "ts-loader": "^7.0.5",
     "ts-node": "^8.10.2",
     "type-fest": "^1.0.2",
+    "typed-emitter": "^1.3.1",
```

Comment:
This package contains only `types` for standard `events`.
What if we add required types directly to our custom event-emitter `common/utils/event-emitter.ts`?

Reply:
Not sure what it would look like ... could you provide an example?

Reply:
Ok, I mean I don't understand purpose of extra types for `events` package when we have our custom event emitter written in Typescript (where we can add any types on purpose). 
Otherwise we could remove `common/utils/event-emitter` and use only standard with these extra-types if it works better for us (to keep usage of single emitter over whole project).